lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java

```diff
@@ -0,0 +1,246 @@
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+    task1 =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassification.asSummary())
+            .workbasketSummary(defaultWorkbasket.asSummary())
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+    task2 =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassification.asSummary())
+            .workbasketSummary(defaultWorkbasket.asSummary())
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+    task3 =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassification.asSummary())
+            .workbasketSummary(defaultWorkbasket.asSummary())
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .buildAndStore(taskService);
+
+    User userWithName = taskanaEngine.getUserService().newUser();
+    userWithName.setId("user-1-1");
+    userWithName.setFirstName("Max");
+    userWithName.setLastName("Mustermann");
+    userWithName.setFullName("Max Mustermann");
+    taskanaEngine.getUserService().createUser(userWithName);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ReturnTaskComments_For_TaskId() throws Exception {
+    TaskComment comment1 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task3.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    TaskComment comment2 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task3.getId())
+            .textField("Text2")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+    TaskComment comment3 =
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task3.getId())
+            .textField("Text3")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService);
+
+    List<TaskComment> taskComments = taskService.getTaskComments(task3.getId());
+
+    assertThat(taskComments).containsExactlyInAnyOrder(comment1, comment2, comment3);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ReturnEmptyList_When_TaskCommentsDontExist() throws Exception {
+    assertThat(taskService.getTaskComments(task2.getId())).isEmpty();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_FailToReturnTaskComments_When_TaskIstNotVisible() {
+    assertThatThrownBy(() -> taskService.getTaskComments(task1.getId()))
+        .isInstanceOf(NotAuthorizedException.class);
+  }
```

Comment:
I think we want to check Exception messages. How about building a NotAuthorizedException expectedException and then asserting equality? :)

Reply:
Ok, I will check the text of the error message